biz.aQute.remote/src/aQute/remote/plugin/RunSessionImpl.java

```diff
@@ -96,8 +97,13 @@ public int launch() throws Exception {
 
 			Agent agent = supervisor.getAgent();
 
-			if (agent.isEnvoy())
+			if (agent.isEnvoy()) {
 				installFramework(agent, dto, properties);
+			} else {
+				for (Entry<String, String> entry : properties.entrySet()) {
+					agent.setSystemProperty(entry.getKey(), entry.getValue());
```

Comment:
Doesn't this just stack up side effects since the system properties are not restored when done?

Reply:
I'm open to other solutions. We'd have to rewrite a testers to get information via some other means. I can't see that happening in short order. This at least is a short term yet pretty safe fix. No one else cares about these properties... and the ONLY way to hand over `-runproperties` in the remote use case.

Reply:
I guess we could track the original value if any and then reset... I can add that.

Reply:
fixed